apps/api/prisma/schema.prisma

```diff
@@ -82,6 +82,7 @@ model Case {
   CaseStatus         CaseStatus[]
   gridReference      GridReference?
   serviceCustomer    ServiceCustomer[]
+  folder             Folder[]
```

Comment:
In the future, absolutely. I was thinking of being able to filter by Folder ID, so that you can be like "whats in this subfolder?" and get all the stuff in there, documents and folders